docs/pod_scenarios.md

```diff
@@ -4,6 +4,7 @@ Kraken consumes [Powerfulseal](https://github.com/powerfulseal/powerfulseal) und
 
 #### Pod chaos scenarios
 Following are the components of Kubernetes/OpenShift for which a basic chaos scenario config exists today. Adding a new pod based scenario is as simple as adding a new config under scenarios directory and defining it in the config.
+For example, for adding a pod level scenario for a custom application, refer to the sample scenario (customapp_pod.yaml) provided in the scenarios directory.
```

Comment:
Can we change this location to be under scenarios/customapp_pod.yaml or a link to the scenario in github like https://github.com/cloud-bulldozer/kraken/blob/master/scenarios/customapp_pod.yml

Reply:
👍